packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -278,6 +286,9 @@ class Auth {
         this.silenceWarnings = config.silenceWarnings || false
 
         this.isPrivate = !!this.clientSecret
+
+        this.callbackURI = callbackURI ? isAbsoluteUrl(callbackURI) 
+        ? callbackURI : `${baseUrl}${callbackURI}` : ''
```

Comment:
Modified this to maintain consistency of the Auth class. Instead of having `passwordlessConfig`, the class will have a `callbackURI` property. If this property is set in the config, then the `mode` will be `callback`.

If the user has enabled passwordless login in their config but no `callbackURI` is set, the `mode` will be assumed to be `sms`.